www/widgets/DemoExampleMaker.js

```diff
@@ -99,8 +103,7 @@ class DemoExampleMaker extends Widget {
         <textarea name="dem-s-text" placeholder="explain step"></textarea>
         <br>
         <button name="dem-preview-step">preview</button>
-        <button name="dem-update-step">update this step</button>
-        <button name="dem-remove-step">remove this step</button>
+        <button id="dem-remove-step" name="dem-remove-step" style="float: right">remove this step</button>
```

Comment:
given the above change, we can now remove the `id="dem-remove-step"` from this element